tests/test_template_loader.py

```diff
@@ -24,7 +24,10 @@ def test_get_dirs__base_dir(self):
             sorted(dirs),
             sorted(
                 [
+                    # Top-level /components dir
                     Path(__file__).parent.resolve() / "components",
+                    # App-level /components dir
+                    Path(__file__).parent.resolve() / "test_app" / "components",
```

Comment:
This is where we test the app-level components dirs. For this to work, I added a Django app to `/tests/test_app` dir